models.py

```diff
@@ -179,21 +179,29 @@ def glimpse_network(images, locs, scope='glimpse_network'):
             # TODO: Lewis implements Glimpse Network here, using also
             # params['g_size'] and params['l_size']
             hidden_sensor_layer = tf.layers.dense(
-                    # not sure this is the correct input
                     inputs = patches,
                     units = params['g_size'],
                     activation = tf.nn.relu)
             
+            dense_sensor_layer = tf.layers.dense(
+                    inputs = hidden_sensor_layer,
```

Comment:
Minor style comment: when passing named arguments into methods like this in Python, I think it's better to not have spaces around the equals sign, i.e. `inputs=hidden_sensor_layer`.  The reason is that, with spaces, you're making an assignment to a variable, which is not quite what's happening here (passing a value with a given name).